packages/edit-widgets/src/filters/move-to-widget-area.js

```diff
@@ -26,17 +36,25 @@ const withMoveToWidgetAreaToolbarItem = createHigherOrderComponent(
 								__internalWidgetId
 						  )
 						: undefined,
+					canInsertBlockInWidgetArea: selectors.canInsertBlockInWidgetArea(
+						props.name
+					),
 				};
 			},
-			[ __internalWidgetId ]
+			[ __internalWidgetId, blockName ]
 		);
 
 		const { moveBlockToWidgetArea } = useDispatch( editWidgetsStore );
+		const hasMultipleWidgetAreas = widgetAreas?.length > 1;
```

Comment:
The standalone editor usually has two widget areas (a proper widget area and inactive widgets)

Not sure if there are themes that won't have a widget area for this to become `false`, but I figured it can't hurt to add this for absolute correctness.

Reply:
There are themes who don't have any widget areas, but in that case we should display a large message that widgets areas are unsupported by the theme. This is what the classic widgets editor does. See #26153